shared-bindings/displayio/Display.c

```diff
@@ -73,32 +76,39 @@
 //|   (b"") are merged together on load. The parens are needed to allow byte literals on subsequent
 //|   lines.
 //|
+//|   The initialization sequence should always leave the display memory access inline with the scan
+//|   of the display to minimize tearing artifacts.
+//|
 //|   :param displayio.FourWire or displayio.ParallelBus display_bus: The bus that the display is connected to
 //|   :param buffer init_sequence: Byte-packed initialization sequence.
 //|   :param int width: Width in pixels
 //|   :param int height: Height in pixels
 //|   :param int colstart: The index if the first visible column
 //|   :param int rowstart: The index if the first visible row
+//|   :param int rotation: The rotation of the display in 90 degree increments
```

Comment:
Just clarify doc for this: are the values 0,1,2,3 or 0,90,180,270?, and clockwise or counterclockwise?

Reply:
Ok, clarified. Thanks for the review!